### PR TITLE
Introduce nodejs kyma installer library for fast-integration tests

### PR DESCRIPTION
**Pull request by aerfio**
**Description**

Changes proposed in this pull request:

- as in title
- installer lib is not yet used anywhere, it'll be used in subsequent PRs

### CODE REVIEW

File: tests/fast-integration/installer/helm.js [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+
+async function helmUninstall(release, namespace) {
+  await execa("helm", ["uninstall", release, "-n", namespace]);
+}
+
+async function helmInstallUpgrade(
+  release,
+  chart,
+  namespace,
+  setValues,
```
Comment (crabtree) [inline tests/fast-integration/installer/helm.js:118]: Please rename `setValues` to just `values`. `set...` may suggest it is a function.

File: tests/fast-integration/installer/helm.js [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+  }
+
+  await execa("helm", args);
+}
+
+async function installRelease(
+  release,
+  namespace = "kyma-system",
+  chart,
+  setValues,
```
Comment (crabtree) [inline tests/fast-integration/installer/helm.js:147]: Same as in `helmInstallUpgrade`

File: tests/fast-integration/installer/helm.js [hunk 3]
```diff
@@ -0,0 +1,381 @@
...
+              console.log(`retrying install of ${release}`);
+              console.log(err);
+            },
+          }
+        );
+      })
+  );
+}
+
+// console.log(
```
Comment (crabtree) [inline tests/fast-integration/installer/helm.js:365]: Leftover?

**Review by crabtree — COMMENTED** [review]
Please see my comments

**Review by crabtree — APPROVED** [review]

**Review by crabtree — DISMISSED** [review]

File: tests/fast-integration/installer/charts/ingress-dns-cert/README.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# Ingress DNS and certificate requests
+
+## Overview
+
+This chart creates certificate request:
```
Comment (kazydek) [inline tests/fast-integration/installer/charts/ingress-dns-cert/README.md:5]:
```suggestion
This chart creates the following certificate request:
```

File: tests/fast-integration/installer/charts/ingress-dns-cert/README.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  namespace: istio-system
+spec:s
+  commonName: '*.{{ .Values.global.domainName }}'
+  secretName: kyma-gateway-certs
+  secretRef:
+    name: kyma-gateway-certs
+    namespace: istio-system
+```
+
+and annotates istio-ingressgateway with:
```
Comment (kazydek) [inline tests/fast-integration/installer/charts/ingress-dns-cert/README.md:21]:
```suggestion
It also annotates `istio-ingressgateway` with:
```

**Review by crabtree — APPROVED** [review]

**Review by kazydek — APPROVED** [review]

**Pull request merged**